content/c/concepts/linked-list/linked-list.md

```diff
@@ -0,0 +1,331 @@
+---
+Title: 'Linked List'
+Description: 'A data structure that contains a collection of nodes that together with the help of pointers, display a sequence.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Singly linked lists'
+  - 'Doubly linked lists'  
+  - 'Circular linked lists'
+  - 'Circular doubly linked lists'
+CatalogContent:
+  - 'learn-c'
+  - 'paths/computer-science'
+---
+
+A **`Linked List`** is a data structure that contains a collection of `nodes` that together with the help of pointers, display a sequence. There are a number of list types, we will cover four of the most common with examples in the `C language`.
+
+## Syntax
+
+```pseudo
+#include 
+#include 
+
+typedef struct Node {
+    void* data;
+    struct Node *next;
+} Node;
+- //push to the front of the list
+struct Node* push(struct Node* head, void* data) {
+  struct Node* new_node = (struct Node*)malloc(sizeof(struct Node));
+  new_node->data = data;
+  new_node->next = head;
+  return new_node;
+}
+int main(){
+  Node* list = NULL;
+-    //remember to set the list after push
+  list = push(list, "!");
+  list = push(list, "World ");
+  list = push(list, "Hello ");
+-    //print out the elements
+  Node* crnt = list;
+  while(crnt != NULL){
+      printf("%s", crnt->data);
+      crnt = crnt->next;
+  }
+  return 0;
+}
+```
+
+## Example
+
+The following C code showcases the creation and traversal of a linked list, which is comprised of a series of nodes with integer values of 10, 20 and 30. The nodes are connected continuously from head to tail displaying the elements of the linked list.
+
+```c
+#include
+#include
```

Comment:
What are these headers? I don't think these are completed please fix this.

Reply:
Hi Dak, thank you for reviewing my latest updates. I went ahead and fixed headers. Please review and let me know if anything else needs review. I went ahead and added along w/ commited and pushed to main branch.

Please advise :)